docs/core/porting/project-structure.md

```diff
@@ -0,0 +1,79 @@
+# Organizing Your Project to Support .NET Framework and .NET Core
+
+This article is to help project owners who want to compile their solution against .NET Framework and .NET Code side-by-side.  It provides several options to organize projects to help developers achieve this goal.
+
+## General Guidance
+
+1.  Create an xproj that targets multiple frameworks.
+    * Scenario is described [here](#replace-existing-projects-with-a-multi-targeted-net-core-project-xproj)
+2.  For projects that use complex logic in their build process, create a Portable Class Library (PCL) that targets .NET Core.
+    * Scenario is described [here](#create-a-portable-class-library-pcl-to-target-net-core)
+
+These are general rules for organizing your project. The scenarios below outline a few variations that can be made based on what fits your repository best.
+
+## Example
+
+Consider the repository below:
+
+![][example-initial-project]
+
+[**Source Code**][example-initial-project-code]
+
+There are several different ways to add support for .NET Core for this repository depending on the constraints and complexity of existing projects which are described below.
+
+### Replace Existing Projects with a Multi-targeted .NET Core Project (xproj)
+
+The repository can be re-organized so that any existing *.csproj are removed and a single *.xproj is created that targets multiple frameworks.  This is a great option because a single project is able to compile for different frameworks.  It also has the power to handle different compilation options, dependencies, etc. per targeted framework.
+
+![][example-xproj]
+
+[**Source Code**][example-xproj-code]
+
+Changes to note are:
+* Addition of `global.json`
+* Replacement of `packages.config` and `*.csproj` with `project.json` and `*.xproj`
+* Changes in the [Car's project.json][example-xproj-projectjson] and its [test project][example-xproj-projectjson-test] to support building for the existing .NET Framework as well as others
+
+### Create a Portable Class Library (PCL) to target .NET Core
+
+If existing projects contain complex build operations or properties in their *.csproj, it may be easier to create a PCL.
+
+![][example-pcl]
+
+[**Source Code**][example-pcl-code]
+
+Changes to note are:
+*  Renaming `project.json` to `{project-name}.project.json`
+    * This prevents potential conflict in Visual Studio when trying to restore packages for the libraries in the same directory
+    * [Documentation is here](https://docs.nuget.org/consume/nuget-faq) under "_I have multiple projects in the same folder, how can I use separate packages.config or project.json files for each project?_"
```

Comment:
Link to the section containing the answer? I.e. https://docs.nuget.org/consume/nuget-faq#working-with-packages.